scripts/pre_push_hook.py

```diff
@@ -295,21 +295,21 @@ def _install_hook():
         print 'Copied file to .git/hooks directory'
 
 
-def does_diff_include_js_files(files_to_lint):
-    """Returns true if diff includes JS files.
+def does_diff_include_ts_files(files_to_lint):
+    """Returns true if diff includes TS files.
 
     Args:
         files_to_lint: list(str). List of files to be linted.
 
     Returns:
-        bool. Status of JS files in diff.
+        bool. Status of TS files in diff.
     """
 
-    js_files_to_check = [
+    ts_files_to_check = [
         filename for filename in files_to_lint if
-        filename.endswith('.js')]
+        filename.endswith('.ts')]
```

Comment:
We should check for both js and ts files since we have some js files present as well:

`filename.endswith('.ts') or filename.endswith('.js')`

Reply:
Ah, I was about to ask this and have left a comment below PTAL!